examples/FedRep/fedrep_CIFAR10_vgg16.yml

```diff
@@ -0,0 +1,80 @@
+
+
+
+clients:
+    # Type
+    type: simple
+
+    # The total number of clients
+    total_clients: 10
+
+    # The number of clients selected in each round
+    per_round: 3
+
+    # Should the clients compute test accuracy locally?
+    do_test: true
+
+    # Whether simulate clients or not
+    simulation: true
+
+    random_seed: 1
+
+server:
+    address: 127.0.0.1
+    port: 8000
+    do_test: false
+    random_seed: 1
+
+    checkpoint_path: models/fedrep/cifar10
+    model_path: models/fedrep/cifar10
+
+data:
+    # The training and testing dataset
+    datasource: CIFAR10
+
+    # Number of samples in each partition
+    partition_size: 600
+
+    # IID or non-IID?
+    sampler: iid
+
```

Comment:
This should be non-iid in order to use the Dirichlet distribution, and the concentration parameter can be a bit lower than 5, say, 1.0.